logging/google/cloud/logging/sink.py

```diff
@@ -116,7 +118,9 @@ def create(self, client=None):
         """
         client = self._require_client(client)
         client.sinks_api.sink_create(
-            self.project, self.name, self.filter_, self.destination)
+            self.project, self.name, self.filter_, self.destination,
+            self._unique_writer_identity
```

Comment:
1. Please use a trailing comma after the last argument
1. Please pass the keyword argument as a keyword (not positional)

Reply:
trailing comma?